src/sentry/mediators/external_requests/issue_link_requester.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+import six
+from uuid import uuid4
+
+from six.moves.urllib.parse import urlparse
+from sentry.http import safe_urlopen, safe_urlread
+from sentry.coreapi import APIError
+from sentry.mediators import Mediator, Param
+from sentry.mediators.external_requests.util import validate
+from sentry.utils.cache import memoize
+from sentry.utils import json
+
+
+class IssueLinkRequester(Mediator):
+    """
+    Makes a request to another service and returns
+    the response.
+    """
+
+    install = Param('sentry.models.SentryAppInstallation')
+    uri = Param(six.string_types)
+    group = Param('sentry.models.Group')
+    fields = Param(object)
+
+    def call(self):
+        self._make_request()
+        return self.response
+
+    def _build_url(self):
+        domain = urlparse(self.sentry_app.webhook_url).netloc
+        url = u'https://{}{}'.format(domain, self.uri)
+        return url
+
+    def _make_request(self):
+        req = safe_urlopen(
+            url=self._build_url(),
+            headers=self._build_headers(),
+            method='POST',
+            data=self.body,
+        )
+
+        body = safe_urlread(req)
+        response = json.loads(body)
```

Comment:
Should you catch errors when JSON fails to parse? Invariably a partner will return HTML pages when they are having an outage, and I'm not sure that should become a sentry error. You may want a datadog metric/log message though to be able to track which partners are having a bad day.